test/disconnect_pin_with_sdc.tcl

```diff
@@ -0,0 +1,27 @@
+# disconnect_pin call for a pin where SDC is defined
+read_liberty asap7_invbuf.lib.gz
+read_liberty asap7_seq.lib.gz
+read_liberty asap7_simple.lib.gz
+read_verilog disconnect_pin_with_sdc.v
```

Comment:
use reg1_asap7.v and asap7_small.lib.gz instead of expanding the required netlists